### PR TITLE
refined storage

### PR DESCRIPTION
**Pull request by bogger33**
This adds:

Quartz Enriched Iron

Cable

Basic Processor

Improved Processor

Advanced Processor

Construction Core

Destruction Core

### CODE REVIEW

File: src/main/java/com/gendeathrow/morechickens/modHelper/RefinedStorageAddon.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import net.minecraft.item.Item;
+
+import com.setycz.chickens.ChickensRegistryItem;
+import com.setycz.chickens.SpawnType;
+
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+public class RefinedStorageAddon extends BaseModAddon{
+
+	@GameRegistry.ObjectHolder("refinedstorage:quartz_enriched_iron")
```
Comment (GenDeathrow) [inline src/main/java/com/gendeathrow/morechickens/modHelper/RefinedStorageAddon.java]: I had no idea this existed... this makes hooking into other mods easy

**Comment by GenDeathrow** [comment]
I'm really iffy about add this one.. I will have to go thur it, I may make some balancing tweaks on it.

**Comment by bogger33** [comment]
That's fine, maybe you can only get the chickens by feeding them a high tier refined storage item?

File: src/main/java/com/gendeathrow/morechickens/modHelper/RefinedStorageAddon.java (hunk 1 above)
Comment (bogger33) [inline src/main/java/com/gendeathrow/morechickens/modHelper/RefinedStorageAddon.java]: It's pretty neat! Someone on the forge IRC channel told me about it

**Pull request merged**